examples/IrisClassificationSignature/iris_handler.py

```diff
@@ -0,0 +1,95 @@
+import logging
+
+import numpy as np
+import torch
+import pandas as pd
+import os
+import json
+from ts.torch_handler.base_handler import BaseHandler
+from mlflow.models.model import Model
+
+logger = logging.getLogger(__name__)
+
+
+class IRISClassifierHandler(BaseHandler):
+    """
+    IRISClassifier handler class. This handler takes an input tensor and
+    output the type of iris based on the input
+    """
+
+    def __init__(self):
+        super(IRISClassifierHandler, self).__init__()
+        self.mlmodel = None
+
+    def preprocess(self, data):
+        """
+        preprocessing step - Reads the input array and converts it to tensor
+
+        :param data: Input to be passed through the layers for prediction
+
+        :return: output - Preprocessed input
+        """
+        from mlflow_torchserve.SignatureValidator import SignatureValidator
+
+        data = json.loads(data[0]["data"].decode("utf-8"))
+        df = pd.DataFrame(data)
+
+        SignatureValidator(model_meta=self.mlmodel)._enforce_schema(
+            df, self.mlmodel.get_input_schema()
+        )
```

Comment:
Does this class do the exact same thing as `mlflow`'s `_enforce_schema`?

Reply:
Yes Harupy. To validate the model signature in mfllow-torchserve plugin we have used the same code.

Reply:
Can we just `_enforce_schema` from `mlflow`?

Reply:
Yes. i have reused the `_enforce_schema` method. I have wrapped all the dependent method into a single class named `SignatureValidator`

Reply:
Created an upstream PR - https://github.com/mlflow/mlflow-torchserve/pull/68 in mlflow-torchserve repo. Please review and let me know your comments.